docs/resources.md

```diff
@@ -162,6 +162,47 @@ spec:
       value: gcr.io/staging-images/kritis
 ```
 
+#### Surfacing the image digest built in a task
+
+To surface the image digest in the output of the `taskRun` the builder tool should produce this information in a [OCI Image Spec](https://github.com/opencontainers/image-spec/blob/master/image-layout.md) `index.json` file. This file should be placed on a location as specified in the task definition under the resource `outputImageDir`.
+
+For example this build-push task defines the `outputImageDir` for the `buildImage` resource in `/worksapce/buildImage`
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: build-push
+spec:
+  inputs:
+    resources:
+    - name: workspace
+      type: git
+  outputs:
+    resources:
+    - name: builtImage
+      type: image
+      outputImageDir: /workspace/builtImage
+  steps: ...
+```
+If no value is specified for `outputImageDir`, it will default to `/tools/image-outputs/{resource-name}`.
```

Comment:
are we using the `/tools` dir for other stuff? if not, what do you think about using `/workspace` instead, to be consistent? (e.g. `/workspace/image-outputs/`)

p.s. if we do use `/tools` think it would be worth adding something to [the container contract docs](https://github.com/tektoncd/pipeline/blob/master/docs/container-contract.md) about this dir, and maybe [our dev docs](https://github.com/tektoncd/pipeline/blob/master/docs/developers/README.md) also?

Reply:
tools was already there used for other things, for example the entrypoint binary. I didn't want to use anything in workspace to avoid conflicting with something the user might be doing

Reply:
It looks like the entrypoint stuff is mounted at [`/builder/tools`](https://github.com/tektoncd/pipeline/blob/master/pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint.go#L40) (I'm not sure when `/builder` became a thing ... 😅 ) And there is a directory created at [`/builder/home`](https://github.com/tektoncd/pipeline/blob/master/pkg/reconciler/v1alpha1/taskrun/resources/pod.go#L72) as well 🤔 

Maybe to be consistent, this could be in `/builder/tools` or something like `/builder/image-outputs`?

(Either way, can you document this in  [the container contract docs](https://github.com/tektoncd/pipeline/blob/master/docs/container-contract.md) about this dir, and maybe [our dev docs](https://github.com/tektoncd/pipeline/blob/master/docs/developers/README.md)?)

Reply:
I think `/builder/home` comes from `knative/build` so it's been here for a while.
I like `/builder/image-outputs` indeed and yeah we should document that — can be done in a follow-up though :wink:

Reply:
will change it to /builder/image-outputs

the `builder` was around from knative build I think

Reply:
added something in the developer readme as well